lib/fastlane/actions/slather.rb

```diff
@@ -32,7 +32,7 @@ def self.run(params)
         command += " --show" if params[:show]
         command += " --source-directory #{params[:source_directory]}" if params[:source_directory]
         command += " --output-directory #{params[:output_directory]}" if params[:output_directory]
-        command += " --ignore #{params[:ignore]}" if params[:ignore]
+        command += " #{params[:ignore].split('|').map { |path| "--ignore \"#{path}\"" }.join(' ')}" if params[:ignore]
```

Comment:
Actually, I'm getting this error after simply feeding array to `ignore` parameter:

```ruby
ignore: ["Pods/*", "../**/*/Xcode*"],
```

```
/Users/maksymgrebenets/.rvm/gems/ruby-2.2.2/gems/fastlane_core-0.32.1/lib/fastlane_core/configuration/config_item.rb:56:in `valid?': 'ignore' value must be a String! Found Array instead. (RuntimeError)
````

Is `:short_option` going to help me with this somehow? Do you have any examples?

Otherwise it looks like I'd have to make changes to `fastlane_core` gem so that `config_item` would take an array as well.